config/doctrine/ChannelPricingLogEntry.orm.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:gedmo="http://gediminasm.org/schemas/orm/doctrine-extensions-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd"
+>
+
+    <mapped-superclass name="Sylius\PriceHistoryPlugin\Model\ChannelPricingLogEntry" table="sylius_price_history_channel_pricing_log_entry">
+        <id name="id" column="id" type="integer">
+            <generator />
+        </id>
```

Comment:
Just wondering if it wouldn't be safer to use uuid instead of an int

Reply:
Agree uuid will be better here but I can't see it used anywhere in Sylius. So I can add it but not sure I'll break some convention or something. And the second question is `ramsey/uuid` (already in our dependencies) or `symfony/uid`?

Reply:
Well, my main concern is with race conditions, I'm not sure if the integer generator strategy can keep up when a lot of logs get created asynchronously in a short timeframe.

Reply:
IMO there is no risk it'll happen. What's more, even if we have UUID we should still have an auto-incremented field to sort log entries in the order in which they were created.

Reply:
Yes, but if two or more log entries are added at the same time, `logged at` is exactly the same.

Reply:
Are we talking about the global state of the table or about the scope of a single `ChannelPricing`?
I don't really see how `logged at` could be exactly the same for the same channel pricing.
What would be the flow for something like that to happen?

Reply:
Two catalog promotions can be applied at the same time or any other customization/integration. Let's say integration with a PIM changes the product's price and the catalog promotion is applied at the same time. With the ID field, we can check if a change made by PIM was first or a change by the catalog promotion applicator.